test/jdk/ProblemList.txt

```diff
@@ -119,6 +119,7 @@ java/awt/Focus/FocusOwnerFrameOnClick/FocusOwnerFrameOnClick.java 8081489 generi
 java/awt/Focus/IconifiedFrameFocusChangeTest/IconifiedFrameFocusChangeTest.java 6849364 generic-all
 java/awt/Focus/AutoRequestFocusTest/AutoRequestFocusToFrontTest.java 6848406 generic-all
 java/awt/Focus/AutoRequestFocusTest/AutoRequestFocusSetVisibleTest.java 6848407 generic-all
+java/awt/Frame/MaximizedToIconified/MaximizedToIconified.java 8296962 macosx-all
```

Comment:
I think 8296962 is the wrong bug ID...

Reply:
oops one digit typo ! thanks for spotting. Fixed in commit just pushed